components/mpas-ocean/cime_config/buildnml

```diff
@@ -893,6 +983,38 @@ if ( -e "$CASEROOT/SourceMods/src.mpaso/$STREAM_NAME" ) {
 	print $stream_file '        output_interval="none">' . "\n";
 	print $stream_file '</stream>' . "\n";
 	print $stream_file '' . "\n";
+	print $stream_file '<stream name="timeSeriesStatsMonthlyMaxRestart"' . "\n";
+	print $stream_file '        type="input;output"' . "\n";
+        if ( $OCN_GRID =~ m/^oRRS1/ ) {
+        	print $stream_file '        io_type="pnetcdf,cdf5"' . "\n";
+        } else {
+		print $stream_file '        io_type="' . "$OCN_PIO_TYPENAME" . '"' . "\n";
```

Comment:
I think we can delete this check for grid.  These are 2-d cell centered fields, so they won't violate size requirements even for the 18to6